sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2155,6 +2155,15 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val ADD_PARTITION_BATCH_SIZE =
+    buildConf("spark.sql.addPartitionInBatch.size")
+      .internal()
+      .doc("The size of partitions to be handled in one turn when use " +
```

Comment:
`The number of partitions` is better than `the size of partitions`. Sometime, `size` means the total bytes of the partitions.